src/x86_64-fma/fft/complex_soa_perm_to_real.py

```diff
@@ -4,7 +4,7 @@
 from common import sqrt2_over_2
 from common import butterfly
 
-import complex_soa
+from .complex_soa import fft4_across_rows
```

Comment:
Please import `complex_soa` instead. Its important to remember the FFT layout (complex SOA) at the call site.